packages/plugin-shiki/src/shiki/rehypePlugin.ts

```diff
@@ -1,17 +1,18 @@
 import { visit } from 'unist-util-visit';
 import type { Plugin } from 'unified';
-import type { Text, Root, ElementContent } from 'hast';
+import type { Element, Text, Root, ElementContent } from 'hast';
 import { fromHtml } from 'hast-util-from-html';
-import type shiki from 'shiki';
+import type { BuiltinTheme, Highlighter } from 'shiki';
 
 interface Options {
-  highlighter: shiki.Highlighter;
+  highlighter: Highlighter;
+  theme: BuiltinTheme;
 }
 
-export const rehypePluginShiki: Plugin<[Options], Root> = function ({
-  highlighter,
-}) {
-  return (tree: Root) => {
+// TODO: migrate to official @shikijs/rehype plugin after upgrading unified/remark/rehype packages
```

Comment:
Do we have any schedule to upgrade `unified` ecosystem packages?

Reply:
https://github.com/web-infra-dev/rspress/discussions/1105#discussioncomment-9507488 In Rspress 2.0 which is in our plan in 2025 Q1-Q2.